jest/babelTransform.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
```

Comment:
The next 3 files were the result of ejecting out of CRA (for purposes of running jest unit tests only) and deleting things that aren't relevant to us.